crates/bevy_ui/src/node_bundles.rs

```diff
@@ -121,10 +119,30 @@ pub struct ImageBundle {
     pub z_index: ZIndex,
 }
 
+impl Default for ImageBundle {
+    fn default() -> Self {
+        Self {
+            node: Default::default(),
+            style: Default::default(),
+            calculated_size: Default::default(),
+            background_color: BackgroundColor(LegacyColor::NONE),
```

Comment:
IMO we could just remove the component from the bundle, as it keeps iso-functionality with the `UiImage::color` tint field